aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -10,8 +10,8 @@ import org.wordpress.aztec.toolbar.AztecToolbar
 import org.wordpress.aztec.toolbar.IAztecToolbarClickListener
 import java.util.*
 
-open class Aztec private constructor(val visualEditor: AztecText, val sourceEditor: SourceViewEditText,
-                                     val toolbar: AztecToolbar, val toolbarClickListener: IAztecToolbarClickListener) {
+open class Aztec private constructor(val visualEditor: AztecText, val toolbar:
+                            AztecToolbar? = null, val toolbarClickListener: IAztecToolbarClickListener? = null) {
```

Comment:
I've realized that it actually doesn't make sense to have a visual editor without a toolbar, does it? If you agree then the `toolbar` and `toolbarClickListener` can actually be made non-nullable (there is no `with()` function that allows this anyway). Sorry for the confusion... 

We can also make `toolbarClickListener` private, since listener getters are not needed.

Nitpick: For readability I think it's better to put the entire parameter declaration on one line (i.e. the `toolbar` parameter)